python/mxnet/kvstore.py

```diff
@@ -613,6 +613,74 @@ def _send_command_to_servers(self, head, body):
         check_call(_LIB.MXKVStoreSendCommmandToServers(
             self.handle, mx_uint(head), c_str(body)))
 
+    def set_server_profiler_config(self, **kwargs):
```

Comment:
I don't think these should be in KVStore class. Its better to put it in mxnet.profiler

Reply:
Ok, will move it today